AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -25,6 +25,12 @@
 #import "ASRangeControllerUpdateRangeProtocol+Beta.h"
 #import "_ASDisplayLayer.h"
 
+typedef NS_ENUM(NSUInteger, ASCollectionViewInvalidationAction) {
+  ASCollectionViewInvalidationActionNone,
+  ASCollectionViewInvalidationActionInvalidateWithoutAnimation,
```

Comment:
Seems pretty cool to have customization here, but these options are not yet self-explanatory.  Add comments, and also consider if we need to include Invalidate twice in the name.